library_conventions/packages.adoc

```diff
@@ -1,8 +1,8 @@
 [#libraryconventions-packages]
 = Package Conventions
-:ipc7351-pdf: http://pcbget.ru/Files/Standarts/IPC_7351.pdf
-:ipc7351-naming-pdf: https://www.pcblibraries.com/downloads/Guidelines!Library-Expert-Land-Pattern-Naming-Convention.asp
-:ipc7351c-slides: https://ocipcdc.org/archive/What_is_New_in_IPC-7351C_03_11_2015.pdf
+:ipc7351-pdf: https://www.ipc.org/TOC/IPC-7351.pdf
```

Comment:
This link only contains the table of contents, but not the actual specification. This is to be expected, because the specification is not available freely :(